README.md

```diff
@@ -1 +1,85 @@
-# node-arguments
\ No newline at end of file
+# Node Arguments
+
+## Overview
+
+This lesson will cover the Node command-line arguments and how to access them.
+
+## Objectives
+
+1. Describe how the CLI argument input to Node script works
+1. Describe an argument array
+2. Describe first arguments
+
+## CLI Arguments
+
+How do we pass data to Node? `node` is just a command-line program and we can pass arguments to it. For example, we can pass two numbers to a program (file `sum.js`) that calculates the sum of them:
```

Comment:
why would you want to pass data to `node`? under what circumstances is that a desirable thing to do? set up the intro to clearly define a situation in which a student would want to pass arguments to node. this sets up the problem, and then letting them know "`node` is just a command-line program and we can pass arguments to it."